### ISSUE TITLE
Changelog AVideo

### ISSUE THREAD

**Issue by gbrn** [WWBN/AVideo#8170]
@DanielnetoDotCom Dani, there's a problem I've been having since I started using AVIDEO. I can't keep track of all the changes.
I know it's not easy to document this, but it's necessary. Even through an organization process for all of us. I've talked about this in the past here, and I've seen other users question it too: #714, #4977, #7338 and others.

It doesn't have to include every change, but it should contain a chronologically ordered list of notable changes for each version, and it shouldn't be a "dump" of a git log "because it doesn't help anyone".

The interesting thing is to create a Changelog.txt file here inside Github itself.
Example: https://github.com/SubtitleEdit/subtitleedit/blob/main/Changelog.txt

For example, the Shorts plugin was added and I didn't hear about it... and I didn't see it anywhere.
![image](https://github.com/WWBN/AVideo/assets/37880073/19cd76ab-60ce-4b02-9726-4927d988d469)
Several plugins are updated and I don't even know...

It would be interesting to document the changes like any other software under development, something like specifying the version and differentiating:
* NEW:
* IMPROVED:
* FIXED:

I think that this documentation will take the application to a higher professional level.

**Comment by para2022** [comment]
Yes this is very frustrating indeed. Things are added and nobody knows they have even been added. Things are changed and nobody knows. A changelog would alow users to debug much more easily. If a change is made to login and then users start having login issues then its obvious of the cause. I have a bug in site that has caused my user count to drop immensely, I'm convinced its an ajax bug in the code as i never did anything to the site for weeks other than updates and suddenly the bug appeared.
